### PR TITLE
Sorted plugin points

### PR DESCRIPTION
**Pull request by aitorvs**
Task/Issue URL: https://app.asana.com/0/488551667048375/1206484244032077/f

### Description
See [asana task](https://app.asana.com/0/488551667048375/1206484244032077/f) and [API proposal](https://app.asana.com/0/488551667048375/1206484244031915)

### Steps to test this PR
Smoke test the app, this is backwards compatible change

### CODE REVIEW

**Comment by aitorvs** [comment]
* `develop`
  * **#4155** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4155?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @aitorvs and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by CDRussell — CHANGES_REQUESTED** [review]
I was testing this branch and the branch above. I noticed that when `@PriorityKey` is omitted, I'm not seeing the plugin show up as expected (I expect to see it after the others which do specify an explicit `PriorityKey`).

Sent screenshots offline to try diagnose, but assuming the issue is in this branch and not the one above.

**Review by CDRussell — APPROVED** [review]
LGTM, but please document at that priority key before merging https://github.com/duckduckgo/Android/pull/4155#discussion_r1478234301

File: anvil/anvil-annotations/src/main/java/com/duckduckgo/anvil/annotations/PriorityKey.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.anvil.annotations
+
+import dagger.MapKey
+
+@MapKey
+annotation class PriorityKey(val priority: Int)
```
[the remark this answers is not in the document]
Comment (aitorvs) [reply anvil/anvil-annotations/src/main/java/com/duckduckgo/anvil/annotations/PriorityKey.kt:31]: done

**Pull request merged**